moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js

```diff
@@ -25,109 +34,117 @@ import { useMutation } from "@apollo/client";
 const ProjectSummaryProjectDescription = ({
   projectId,
   data,
+  refetch,
   classes,
   handleSnackbar,
   listViewQuery,
 }) => {
   const originalDescription =
     data?.moped_project?.[0]?.project_description ?? null;
 
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+  } = useForm({
+    defaultValues: { description: originalDescription },
+    mode: "onChange",
+    resolver: yupResolver(validationSchema),
+  });
+
   const [editMode, setEditMode] = useState(false);
-  const [description, setDescription] = useState(originalDescription);
 
-  const [updateProjectDescription] = useMutation(PROJECT_UPDATE_DESCRIPTION);
+  const [updateProjectDescription, { loading }] = useMutation(
+    PROJECT_UPDATE_DESCRIPTION
+  );
 
   /**
-   * Resets the project description to original value
+   * Switch to view mode on close
    */
   const handleProjectDescriptionClose = () => {
-    setDescription(originalDescription);
     setEditMode(false);
   };
 
   /**
    * Saves the new project description...
    */
-  const handleProjectDescriptionSave = () => {
-    if (description.trim().length > 0) {
-      updateProjectDescription({
-        variables: {
-          projectId: projectId,
-          description: description,
-        },
-        refetchQueries: [{ query: listViewQuery }, "ProjectSummary"],
```

Comment:
The refetch of the project summary data used to be handled here, but I moved it into the save handler so the refetch happens before the edit mode switches back to the view mode.